PIL/ImageShow.py

```diff
@@ -69,8 +69,9 @@ def show(self, image, **options):
             # FIXME: auto-contrast if max() > 255?
         else:
             base = Image.getmodebase(image.mode)
-        if base != image.mode and image.mode != "1":
+        if base != image.mode and image.mode != "1" and image.mode != "RGBA":
             image = image.convert(base)
```

Comment:
That's reasonably easy to test - I can just copy the bmp files to a windows machine and see. I'll try that sometimes.